go.mod

```diff
@@ -11,58 +11,74 @@ exclude github.com/hashicorp/consul/api v1.1.0
 // to fix: [CVE-2023-32731] CWE-Other
 replace google.golang.org/grpc => google.golang.org/grpc v1.55.0
 
+// [CVE-2024-24786] CWE-835: Loop with Unreachable Exit Condition ('Infinite Loop')
+replace google.golang.org/protobuf => google.golang.org/protobuf v1.33.0
+
+// [CVE-2023-48795] CWE-354: Improper Validation of Integrity Check Value
+replace golang.org/x/crypto => golang.org/x/crypto v0.17.0
+
```

Comment:
Please remove `replace` directives in production as they bypass hash verification and create a supply chain security vulnerability.